src/Controls/src/Core/Platform/Windows/CollectionView/GroupedItemTemplateCollection.cs

```diff
@@ -15,6 +14,10 @@ internal class GroupedItemTemplateCollection : ObservableCollection<GroupTemplat
 		readonly BindableObject _container;
 		readonly IMauiContext _mauiContext;
 		readonly IList _groupList;
+		readonly NotifyCollectionChangedEventHandler _collectionChanged;
+		readonly WeakNotifyCollectionChangedProxy _proxy = new();
+
+		~GroupedItemTemplateCollection() => _proxy.Unsubscribe();
```

Comment:
Can we add `GroupedItemTemplateCollection` to a test like this one:

https://github.com/dotnet/maui/blob/2a9f9fd1980d521d1a36f72febc5ad9f94dc706e/src/Controls/tests/Core.UnitTests/WeakEventProxyTests.cs#L12-L30